content/_components/shopify-admin-v2/index.md

```diff
@@ -482,12 +474,12 @@ Lookup a single object by its ID - only query with one argument `id` can be used
 #### Configuration Fields
 
 * **Object Type** - (dropdown, required): Object-type to lookup on. E.g `Customer`.
-* **Select basic fields for resulting object** - (dropdown, optional): Here provided only basic fields that can be included in resulting object, it may affect on query cost
-* **You can provide additional fields here** - (string, optional): Resulting object can be expanded using GraphQL request, it may affect on query cost
+* **Select basic fields for resulting object** - (dropdown, optional): Here provided only basic fields that can be included in resulting object, it may affect on query cost.
```

Comment:
* **Select basic fields for resulting object** - (dropdown, optional): Here only basic fields can be included in the resulting object, it may affect query cost.